packages/neon_framework/lib/src/storage/request_cache.dart

```diff
@@ -159,53 +146,52 @@ WHERE account = ? AND key = ?
           {
             'account': account.id,
             'key': key,
-            'value': value,
-            'etag': parameters?.etag,
-            'expires': parameters?.expires?.secondsSinceEpoch,
+            'statusCode': response.statusCode,
+            'body': response.bodyBytes,
+            'headers': json.encode(response.headers),
```

Comment:
we are encoding the headers two times.
Once here and once for the insert statement.

Please reduce this by storing it in a variable.